lib/tasks/after_tasks.rake

```diff
@@ -582,6 +582,21 @@ namespace :After do
     end
   end
 
+  desc "Enforce HTTPS where available for embedded media from ning.com and vidders.net"
+  task(enforce_https: :environment) do
+    Chapter.find_each do |chapter|
+      puts chapter.id if (chapter.id % 1000).zero?
+      if chapter.content.match /<(embed|iframe).*(ning.com|vidders.net)/
```

Comment:
You should escape the dots in `ning.com|vidders.net` or they'll match any character.

Even with that fix, there can still be false matches (e.g. works can link to `vidders.net` outside the embed tags), but I think this is a good enough heuristic.

Reply:
The <(embed|iframe) is still in my expression, and my goal was to match <(embed or iframe)(any characters)(ning.com or vidders.net). I checked it with a regex tester (https://rubular.com/) and it doesn't match if I replace embed with something else (other than iframe, which maybe isn't necessary here, but I figured it wouldn't do any harm). It does match embed tags that only contain ning.com or vidders.net and not both, but I don't know how likely it is for those to appear, and if they should be ignored if they do.

But yeah, I did escape the dots.